packages/healthcheck/README.md

```diff
@@ -0,0 +1,11 @@
+# Usage
+
+```
+const healthChecks = {
+  service1: 'https://service1.com/ping',
+  service2: { method: 'POST', url: 'https://service2.com/ping' },
+  service3: { url: 'https://service3.com/ping', options: { headers: { 'Authorization': `Basic dXNlcjpwYXNz` } } },
+};
+
+app.use(require('@maxdome/healthcheck')({ healthChecks }));
```

Comment:
The router defines "/" and "/details", would it not better to use it with a prefix like "/healthcheck" in the example?

Reply:
You're right. I was about to use it like so:
```javascript
app.use('/health', require('@maxdome/healthcheck')({ healthChecks }));
```

I will update `README.md` but we can as well force `/health` and `/health/details` in the package.